src/models/transaction.ts

```diff
@@ -1,13 +1,11 @@
 export class Transaction {
   chain: string
   txHash: string
-  blockNumber: number
-  blockTimestamp: number
+  blockNumber: bigint
```

Comment:
Mind if I ask you what happened to the timestamp?

Reply:
It's not part of the transaction receipt from viem. Still need to figure out if there's a way for me to get that. Possibly by grabbing the block.